pkg/sac/testconsts/sac_consts.go

```diff
@@ -2,11 +2,10 @@ package testconsts
 
 // Clusters and Namespaces for scoped access control tests
 const (
-	Cluster1     = "aaaaaaaa-bbbb-4011-0000-111111111111"
```

Comment:
I'm trying to understand what is the reason behind changing values from UUIDs to a simple string.

Reply:
@mtodor I think he needs a rebase since I pushed my changes yesterday.  That would solve this diff and probably clean up the review in general.